modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/service/test/SegmentsExperienceLocalServiceTest.java

```diff
@@ -658,33 +709,29 @@ public void testUpdateSegmentsExperienceToZeroWithBelowSegmentsExperiences()
 				_group.getGroupId(), _classNameId, _classPK);
 
 		Assert.assertEquals(
-			segmentsExperience1.getPriority(),
-			SegmentsExperienceConstants.PRIORITY_DEFAULT - 1);
+			_PRIORITY_DEFAULT - 1, segmentsExperience1.getPriority());
 
 		SegmentsExperience segmentsExperience2 =
 			SegmentsTestUtil.addSegmentsExperience(
 				_group.getGroupId(), _classNameId, _classPK);
 
 		Assert.assertEquals(
-			segmentsExperience2.getPriority(),
-			SegmentsExperienceConstants.PRIORITY_DEFAULT - 2);
+			_PRIORITY_DEFAULT - 2, segmentsExperience2.getPriority());
 
 		segmentsExperience1 =
 			_segmentsExperienceLocalService.updateSegmentsExperiencePriority(
 				segmentsExperience1.getSegmentsExperienceId(),
 				segmentsExperience1.getPriority() + 1);
 
 		Assert.assertEquals(
-			SegmentsExperienceConstants.PRIORITY_DEFAULT + 1,
-			segmentsExperience1.getPriority());
+			_PRIORITY_DEFAULT, segmentsExperience1.getPriority());
 
 		segmentsExperience2 =
 			_segmentsExperienceLocalService.fetchSegmentsExperience(
 				segmentsExperience2.getSegmentsExperienceId());
 
 		Assert.assertEquals(
-			SegmentsExperienceConstants.PRIORITY_DEFAULT - 1,
-			segmentsExperience2.getPriority());
+			_PRIORITY_DEFAULT - 2, segmentsExperience2.getPriority());
```

Comment:
The segmentsExperience2 priorityh should be -1, right?, once segmentsExperience1 has ben moved to the 0 priority.

Reply:
IMHO no. When segmentsExperience1 is moved to the 0 priority it becomes in the default SegmentExperience and, the experience that was the default SegmentExperience before becomes in the SegmentExperience with -1 priority. So segmentsExperience2 remains with -2 priority.

Reply:
That is totally right :), I will mark it as technical debt, so we will add it the default experience priority to the test too. Thanks a lot :)